### ISSUE TITLE
how come only surrogate generator loss is computed?

### ISSUE THREAD

**Issue by mjdietzx** [jiamings/cramer-gan#2]
Hey, great job! I've only skimmed through the paper, but just curious why generator loss is commented out in your implementation? https://github.com/jiamings/cramer-gan/blob/master/cramer-gan.py#L37

**Comment by jiamings** [comment]
I did not look into in too much detail either, but if the generator loss is updated, why compute the surrogate loss for discriminator? It seems to me a waste of computation resources...

**Comment by slowbull** [comment]
In the paper "we use to define a loss Lsurrogate (X, Y ) similar to (10) with which we train the critic. In practice, it works well to train also the generator to minimize the Lsurrogate loss." it looks like both L_g and L_surrogate  can be used as the loss function for the generator.

**Comment by jiamings** [comment]
Thanks @slowbull ! Then it seems to me that the current code for surrogate loss is fine. 
It is also okay to use the actual generator loss.
